### PR TITLE
antisnake

### PR DESCRIPTION
**Pull request by vsupruniuk**
- [DEMO LINK](https://vsupruniuk.github.io/layout_antisnake/)
- [TEST REPORT LINK](https://vsupruniuk.github.io/layout_antisnake/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/snake.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #fff;
+    height: 300px;
+  }
+
+  &__child--1 {
+    background-color: rgb(100%, 0%, 0%);
+  }
```
Comment (stanislav-topikha) [inline src/styles/blocks/snake.scss:19]: You can put modifiers into the element

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
Good job! But there is no test report.

**Review by stanislav-topikha — APPROVED** [review]

File: src/styles/blocks/snake.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
```
Comment (Olexandr6) [inline src/styles/blocks/snake.scss:3]: 🔥

File: src/styles/blocks/snake.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  @media (min-width: 900px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  &__child {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #fff;
+    height: 300px;
```
Comment (Olexandr6) [inline src/styles/blocks/snake.scss:14]: You use 300px in several places, so let's create an`scss` variable for this.

File: src/styles/blocks/snake.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      background-color: rgb(40%, 0%, 0%);
+    }
+
+    &--5 {
+      background-color: rgb(20%, 0%, 0%);
+    }
+
+    &--6 {
+      background-color: rgb(0%, 0%, 0%);
+    }
```
Comment (Olexandr6) [inline src/styles/blocks/snake.scss:38]: Let's find another solution for this logic. Don't create additional classes. Here you have almost the same styles for every class.

Hint: use `nth-child: ` + `@for` cylce

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Your solution is good, but can be improved; Let's do this🔥

File: src/styles/blocks/snake.scss [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  &__child {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #fff;
+    height: $min-width-square;
+
+    @for $var from 1 through 6 {
+      &--#{$var} {
+        background: mix(#f00, #000, 100 - 20 * ($var - 1));
```
Comment (Olexandr6) [inline src/styles/blocks/snake.scss:18]: 🔥

**Review by Olexandr6 — APPROVED** [review]
Great fob🔥

**Pull request closed without merging**